admin/view/wp-slimstat-reports.php

```
@@ -1553,6 +1606,281 @@ public static function show_events($_args = [])
+            if (count($funnels) > 1 && !$is_widget) {
+                echo '<div class="slimstat-funnel-tabs">';
+                foreach ($funnels as $i => $f) {
+                    $active_class = ($i === 0) ? ' active' : '';
+                    echo '<button class="button slimstat-funnel-tab' . $active_class . '" data-funnel-index="' . $i . '">' . esc_html($f['name']) . '</button>';
+                }
+                echo '</div>';
+            }
+
+            foreach ($funnels as $fi => $funnel) {
+                // Only compute the active (first) funnel; others load via tab switch
+                $step_results = ($fi === 0) ? wp_slimstat_db::get_funnel_results($funnel) : [];
```

PHPStan (2.1.46)

[error] Call to an undefined static method wp_slimstat_db::get_funnel_results().


(staticMethod.notFound)